tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Builders/AddressBuilder.cs

```diff
@@ -0,0 +1,84 @@
+﻿using NHSD.BuyingCatalogue.Organisations.Api.Models;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.UnitTests.Builders
+{
+    internal sealed class AddressBuilder
+    {
+        private string _line1;
+        private string _line2;
+        private string _line3;
+        private string _line4;
+        private string _town;
+        private string _county;
+        private string _postcode;
+        private string _country;
+
```

Comment:
Do you think it would be a good idea to add some sensible defaults? At the moment in the tests you are having to add every single property.